src/modules/welcome/Config.cpp

```diff
@@ -261,13 +268,47 @@ QString Config::supportUrl() const
 void Config::setSupportUrl(const QString& url)
 {
     m_supportUrl = url;
+    emit supportUrlChanged();
 }
 
+void RequirementsModel::retranslate()
+{
+    if ( !m_satisfiedRequirements )
+    {
+        QString message;
+        const bool setup = Calamares::Settings::instance()->isSetupMode();
 
+        if ( !m_satisfiedMandatory )
+        {
+            message = setup ? tr( "This computer does not satisfy the minimum "
+            "requirements for setting up %1.<br/>"
+            "Setup cannot continue. "
+            "<a href=\"#details\">Details...</a>" )
+            : tr( "This computer does not satisfy the minimum "
+            "requirements for installing %1.<br/>"
+            "Installation cannot continue. "
+            "<a href=\"#details\">Details...</a>" );
+
+        }else
```

Comment:
Please apply `ci/calamaresstyle`. You'll need astyle and clang-format (version 7 or later) installed for that.